starter-code/src/clue.js

```diff
@@ -92,3 +92,252 @@ var roomsArray = [];
+}
+    
+charactersArray.push(missScarlet);
+
+var mrsPeacock = {
+    first_name:   "Eleanor",
+    last_name:    "Peacock",
+    color:        "blue",
+    description:  "She is from a wealthy family and uses her status and money to earn popularity",
+    age:          36,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation:   "Socialité"
+}
+
+charactersArray.push(mrsPeacock);
+
+var mrMustard = {
+    first_name:   "Jack",
+    last_name:    "Mustard",
+    color:        "yellow",
+    description:  "He is a former football player who tries to get by on his former glory",
+    age:          62,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:   "Retired Football player",
+}
+
+charactersArray.push(mrMustard);
+
+var rope = {
+    name: "rope",
+    weight: 10
+  }
+  
+  weaponsArray.push(rope);
+
+var knife = {
+    name: "knife",
+    weight: 8
+  }
+
+  weaponsArray.push(knife);
+
+var candlestick = {
+    name: "candlestick",
+    weight: 2
+  }  
+
+  weaponsArray.push(candlestick);
+
+var dumbbell = {
+    name: "dumbell",
+    weight: 30
+  }  
+
+  weaponsArray.push(dumbbell);
+
+var poison  = {
+    name:   "poison",
+    weight: 2
+  }  
+
+  weaponsArray.push(poison);
+
+var axe = {
+    name: "axe",
+    weight: 15
+  }  
+
+  weaponsArray.push(axe);
+
+var bat = {
+    name:   "bat",
+    weight: 13
+  }  
+
+  weaponsArray.push(bat);
+
+var trophy  = {
+    name:   "trophy",
+    weight: 25
+  }  
+
+  weaponsArray.push(trophy);
+
+var pistol = {
+    name:   "pistol",
+    weight: 20
+  }
+
+  weaponsArray.push(pistol);
+
+  var room1 = {
+      name: "Dinning Room"
+    }
+
+    roomsArray.push(room1);
+
+  var room2 = {
+        name: "Conservatory"
+    }
+
+    roomsArray.push(room2);
+  
+  var room3 = {
+      name: "Kitchen"
+  }
+  
+  roomsArray.push(room3);
+
+  var room4 = {
+      name: "Study"
+  }
+
+  roomsArray.push(room4);
+
+  var room5 = {
+        name: "Library"
+  }
+
+  roomsArray.push(room5);
+
+  var room6 = {
+        name: "Billiard Room"
+  }
+
+  roomsArray.push(room6);
+
+  var room7 = {
+        name: "Lounge"
+  }
+
+  roomsArray.push(room7);
+
+  var room8 = {
+        name: "Ballroom"
+  }
+
+  roomsArray.push(room8);
+
+  var room9 = {
+        name: "Hall"
+  }
+
+  roomsArray.push(room9);
+
+  var room10 = {
+        name: "Spa"
+  }
+
+  roomsArray.push(room10);
+
+  var room11 = {
+        name: "Living Room"
+  }
+
+  roomsArray.push(room11);
+
+  var room12 = {
+        name: "Observatory"
+  }
+
+  roomsArray.push(room12);
+
+  var room13 = {
+        name: "Theater",
+  }
+
+  roomsArray.push(room13);
+
+  var room14 = {
+      name: "Guest House"
+  }
+
+  roomsArray.push(room14);
+
+  var room15 = {
+        name: "Patio"
+  }
+
+  roomsArray.push(room15);
+
+function randomSelector(aArray){
+    if (aArray.length!==0){
+    return aArray[Math.floor(Math.random()*aArray.length)];
```

Comment:
```
return aArray[Math.floor(Math.random()*(aArray.length + 1))];
```
Otherwise it will not be able to pick randomly the last item.

Reply:
Oops! Bad correction! If you are using the number as an index it's okay :disappointed: sorry